src/library/scala/math/BigDecimal.scala

```diff
@@ -498,10 +498,10 @@ extends ScalaNumber with ScalaNumericConversions with Serializable with Ordered[
   /** Division and Remainder - returns tuple containing the result of
    *  divideToIntegralValue and the remainder.  The computation is exact: no rounding is applied.
    */
-  def /% (that: BigDecimal): (BigDecimal, BigDecimal) =
-    this.bigDecimal.divideAndRemainder(that.bigDecimal, mc) match {
-      case Array(q, r)  => (new BigDecimal(q, mc), new BigDecimal(r, mc))
```

Comment:
Just to understand: was this the one and only `cased Array(...)` pattern in the standard library?

Reply:
I am sure there are more in the tests (which we don't compile yet with Dotty), but yes it seems it was the only one in the standard library